app/experimenter/nimbus-ui/src/components/TableAudience/index.tsx

```diff
@@ -55,15 +55,26 @@ const TableAudience = ({ experiment }: TableAudienceProps) => {
           </tr>
         )}
         {experiment.targetingConfigSlug && (
-          <tr>
-            <th>Custom audience</th>
-            <td data-testid="experiment-target">
-              {displayConfigLabelOrNotSet(
-                experiment.targetingConfigSlug,
-                targetingConfigSlug,
-              )}
-            </td>
-          </tr>
+          <>
+            <tr>
+              <th>Custom audience</th>
+              <td data-testid="experiment-target">
+                {displayConfigLabelOrNotSet(
+                  experiment.targetingConfigSlug,
+                  targetingConfigSlug,
+                )}
+              </td>
+            </tr>
+            <tr>
+              <th>Full targeting expression</th>
+              <td
+                data-testid="experiment-target-expression"
+                className="text-monospace"
+              >
+                {experiment.targetingConfigTargeting}
+              </td>
+            </tr>
+          </>
```

Comment:
Is a targeting config slug of "No Targeting" the same as not having a targeting config? If so, maybe we should hide the row since it yields no expression?

![Screen Shot 2021-01-14 at 10 23 01 AM](https://user-images.githubusercontent.com/6392049/104611604-4f70d100-5653-11eb-9853-af77173225e2.png)

Reply:
Yeah, no targeting == no expression. Probably good idea to hide the row :+1: - will toss that tweak in there before merging